deployment/cdk/opensearch-service-migration/dp_pipeline_template.yaml

```diff
@@ -1,18 +1,36 @@
+# Name of the Data Prepper pipeline
 historical-data-migration:
+  # Source cluster configuration
   source:
     opensearch:
+      # DO NOT CHANGE the hosts value - this will be updated by the CDK
```

Comment:
I would add a note that they would need to change this only if running locally (in Docker)

Reply:
Updated comment to:

```
# CDK code will replace this value, so DO NOT CHANGE this
# unless the file is being used outside of the CDK
```